src/components/CreateCoursePage/index.jsx

```diff
@@ -10,6 +10,7 @@ import ConfirmationModal from '../ConfirmationModal';
 
 import { formatPriceData } from '../../utils';
 
+const typeWhiteList = process.env.TYPE_WHITELIST ? process.env.TYPE_WHITELIST.split(',') : [];
```

Comment:
A toplevel variable? Alright.

Reply:
Otherwise this would get calculated much more often and this is just less calls. Basically copying the way the other whitelist was created.